monitoring.tf

```diff
@@ -0,0 +1,196 @@
+# Base infra logging
+# Deploy fluentbit with the fluent-operator and configure it so that pods with the ${var.logging_annontation} annotation have
+# they logs pushed to CloudWatch.
+# By default (hardcoded), fluent-operator and fluent-bit will have this annotation set
+locals {
+  # Namespace for the resources deployed by the fluent-operator (fluent-bit will be here too)
+  fluent_namespace                       = "monitoring"
+  fluentbit_cloudwatch_log_group         = "/${local.stack_name}/fluentbit-logs"
+  fluentbit_cloudwatch_log_stream_prefix = "."
```

Comment:
make this meaningful like infra or something. 

In the next update, we should add template functions here and then the logs streams are nice and easy to read